src/query/plan/preprocess.cpp

```diff
@@ -584,6 +584,10 @@ std::vector<SingleQueryPart> CollectSingleQueryParts(SymbolTable &symbol_table,
       if (auto *merge = utils::Downcast<query::Merge>(clause)) {
         query_part->merge_matching.emplace_back(Matching{});
         AddMatching({merge->pattern_}, nullptr, symbol_table, storage, query_part->merge_matching.back());
+      } else if (auto *call_subquery = utils::Downcast<query::CallSubquery>(clause)) {
+        auto subquery =
+            std::make_shared<QueryParts>(CollectQueryParts(symbol_table, storage, call_subquery->cypher_query_));
```

Comment:
What you can try but I am really not sure if it will work is:
```
query_part->subqueries.emplace_back(std::make_shared<QueryParts>(symbol_table, storage, call_subquery->cypher_query_));

```

Reply:
works